site/src/pages/WorkspacePage/BuildRow.tsx

```diff
@@ -58,7 +58,7 @@ export const BuildRow: React.FC<BuildRowProps> = ({ build }) => {
               >
                 <span>
                   <strong>{initiatedBy}</strong>{" "}
-                  {build.reason !== "initiator" ? "automatically" : ""}
+                  {build.reason !== "initiator" ? "automatically " : ""}
```

Comment:
wondering if there is a better way to make this space more explicit lol

Reply:
eh 🤷‍♀️ as long as it's tested (which it is now)